lambdas/gro-convert-to-json/src/test/java/uk/gov/di/data/lep/GroConvertToJsonTest.java

```diff
@@ -142,6 +144,35 @@ class GroConvertToJsonTest {
         "<DeathRegistrationGroup xmlns=\"http://www.ons.gov.uk/gro/OGDDeathExtractDWP\" xmlns:ns1=\"http://www.govtalk.gov.uk/people/PersonDescriptives\" xmlns:ns2=\"http://www.govtalk.gov.uk/people/AddressAndPersonalDetails\" xmlns:ns3=\"http://www.ons.gov.uk/gro/people/GROAddressDescriptives\" xmlns:ns4=\"http://www.ons.gov.uk/gro/people/GROPersonDescriptives\" xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xsi:schemaLocation=\"http://www.ons.gov.uk/gro/OGDDeathExtractDWP .\\OGDDeathExtractDWP-v1-1.xsd\">" +
         "<RecordCount>6</RecordCount>" +
         "</DeathRegistrationGroup>";
+    private static final String mockS3objectResponseExpectedRecordCountNotMatchingFoundRecordCount =
```

Comment:
Can we use the
```java
private static final String X =  """
    <XmlGoesHere></XmlGoesHere>
    """;
``` 
format for these big blobs of XML please? I think it makes it a bit cleaner/easier to understand